### PR TITLE
Jacob Sacks - API Project

### PR DESCRIPTION
**Pull request by jsacks2**
This is the first draft of my API assignment. I hope it's good 🤞

### CODE REVIEW

File: student-work/2023/Jacob-Sacks/API-final-project/get.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+This HTTP request retrieves the bill for a specified table. The response is sent to the server to be brought the customer. The request takes the table number and outputs an itemized bill. For more information about the API in general, see the [readme file](readme.md).
+
+## URL
+```
+https://api.gpmd.com/tableNo?id=<NUMBER>
+```
+
+Where \<NUMBER> is the customer's table number
+
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Jacob-Sacks/API-final-project/get.md:13]:
```suggestion
For takeout orders the table number is 99.
```

File: student-work/2023/Jacob-Sacks/API-final-project/get.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+| timestamp   | Date        | Time order was placed                              |
+| item        | JSON object | Numbered list of items ordered                                       |
+| itemOrdered | JSON object | Details of item                                    |
+| type        | String      | Type of meal<br>(for POC, there's only burgerMeal) |
+| cost        | Float       | Cost of item                                       |
+
+## Example request
+
+This would be the request for the bill for table number 18:
+```
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Jacob-Sacks/API-final-project/get.md:38]:
```suggestion

``` none
```

File: student-work/2023/Jacob-Sacks/API-final-project/get.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+curl -X GET "http://URL/tableNo?id=18"
+```
+
+## Example response
+
+```
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Jacob-Sacks/API-final-project/get.md:44]:
```suggestion
``` json
```

File: student-work/2023/Jacob-Sacks/API-final-project/get.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+}
+```
+
+## HTTP status codes
+
+Here are the possible HTTP responses:
+
+| Status code | Description    | Description                                            |
+|-------------|----------------|--------------------------------------------------------|
+| 200         | Success        | Bill was successfully printed |
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Jacob-Sacks/API-final-project/get.md:69]:
```suggestion
| 200         | Success        | Request for the bill was successfully executed|
```

File: student-work/2023/Jacob-Sacks/API-final-project/post.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+| type     | Type of drink | string    | coke, cokeZero, sprite, spriteZero,<br>lemonade, icedTea, seltzer, none | none    | N         |
+| size     | Size of drink | string    | small, medium, large, none                                              | none    | N         |
+| ice      | Ice in drink  | boolean   | true or false                                                           | true    | N         |
+
+## Example request
+
+Here is an example of a POST request for a customer's order:
+```
+curl -H "Content-Type: application/json" -X POST -d'
+```
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Jacob-Sacks/API-final-project/post.md:57]:
```suggestion
```

```

File: student-work/2023/Jacob-Sacks/API-final-project/post.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+```
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Jacob-Sacks/API-final-project/post.md:58]:
```suggestion
``` json
```

File: student-work/2023/Jacob-Sacks/API-final-project/post.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+}
+```
+
+## HTTP status codes
+
+Here are the possible HTTP responses:
+
+| Status code | Description    | Description                                            |
+|-------------|----------------|--------------------------------------------------------|
+| 200         | Success        | Meal was successfully received and sent to the kitchen |
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Jacob-Sacks/API-final-project/post.md:98]:
```suggestion
| 200         | Success        | Meal request was successfully received and sent to the kitchen |
```

**Review by Laura-Novich-OBW — COMMENTED** [review]
Fantastic start.
small comments. 
Please apply or reject

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
see comments within

**Comment by jsacks2** [comment]
I just committed the suggested changes. Thanks for the quick feedback.

**Review by Laura-Novich-OBW — APPROVED** [review]

**Pull request merged**